modules/affile/affile-dest.c

```diff
@@ -292,24 +292,28 @@ static void
 affile_dw_set_owner(AFFileDestWriter *self, AFFileDestDriver *owner)
 {
   GlobalConfig *cfg = log_pipe_get_config(&owner->super.super.super);
+  log_pipe_set_config(&self->super, cfg);
```

Comment:
This does not fix the NULL pointer deref issue as `log_pipe_get_config` dereferences `owner`.

Reply:
You are right, I messed it up during the rebase :facepalm:  .